react/features/base/participants/functions.js

```diff
@@ -123,6 +123,18 @@ export function getParticipantById(
     return remote.get(id) || (local?.id === id ? local : undefined);
 }
 
+/**
+ * Returns a selector that gets the participant by id.
+ *
+ * @param {string} id - The ID of the participant to retrieve.
+ * @returns {Function}
+ */
+export function getParticipantByIdSelectorCreator(id: string) {
+    return function(state: Object | Function) {
+        return getParticipantById(state, id);
```

Comment:
i wonder if it would be more useful for future implementations to create a wrapper hook over useSelector that passes through params after state

Reply:
Yep, good ideea. I'll create one. It would also be great if all selectors would have the similar signature with the `state` as the first param. At least from now on we can try to enforce this convention.